CONTRIBUTING.md

```diff
@@ -0,0 +1,21 @@
+# Contributing
```

Comment:
I think we should have the same contributing file across charms. [This is the one for the `smf`](https://github.com/canonical/sdcore-smf-operator/blob/08f720ed7c26042bedcf1c5a72a98836436054aa/CONTRIBUTING.md) based on the `nrf` one but adding the `tox -e fmt`.

Reply:
Done